repositories/geometry2.BUILD.bazel

```diff
@@ -117,3 +117,18 @@ ros2_cpp_binary(
         "@ros2_rclcpp//:rclcpp_components",
     ],
 )
+
+ros2_cpp_library(
+    name = "cpp_tf2_sensor_msgs",
+    srcs = [
+        "tf2_sensor_msgs/include/tf2_sensor_msgs/tf2_sensor_msgs.hpp",
+    ],
+    includes = ["tf2_sensor_msgs/include"],
+    visibility = ["//visibility:public"],
+    deps = [
+        ":tf2",
+        ":tf2_ros",
+        "@eigen",  # TODO we could make the user responsible for providing eigen and remove it here
```

Comment:
Just copy and use https://github.com/mvukov/rules_ros/blob/main/repositories/eigen.BUILD.bazel and https://github.com/mvukov/rules_ros/blob/main/repositories/repositories.bzl#L204-L211 (if there is a newer version, please use that one).

Reply:
Done